<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1630</width>
    <height>1093</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="HysteresisChart" name="hysteresis_chart" native="true"/>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="MainMenuWindow" name="widget_main_menu" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="BHChart" name="B_H_chart" native="true"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="BarkhausenChart" name="barkhausen_chart" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1630</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainMenuWindow</class>
   <extends>QWidget</extends>
   <header location="global">mainmenuwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BarkhausenChart</class>
   <extends>QWidget</extends>
   <header location="global">barkhausen_chart.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HysteresisChart</class>
   <extends>QWidget</extends>
   <header location="global">hysteresis_chart.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BHChart</class>
   <extends>QWidget</extends>
   <header location="global">B_H_chart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
